src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.TodoCommand;
+
+/**
+ * Parser class allows program to parse user commands.
+ */
+public class Parser {
+    Parser() {}
+
+    /**
+     * Primary parse method that takes in a String command and returns a Command object that represents it.
+     * @param command User command to be parsed
```

Comment:
Remember to leave "Empty line between description and parameter section"!

Reply:
Oh right, noticed my mistake when reviewing someone else too xD